modules/rte_mis_reimbursement/src/Services/RteReimbursementHelper.php

```diff
@@ -181,7 +183,51 @@ public function loadStudentData(?string $school_id = NULL, ?string $academic_yea
         $rows['field_medium'] = $student_medium;
         // School Tution Fee.
         $rows['school_tution_fee'] = $this->schoolTutionDetails($current_user_fee_details, $student_gender, $student_medium, $class_value) ?? 0;
-        $government_tution_fees = $government_fee['tution_fee'] ?? 0;
+        // Set default value.
+        $government_tution_fees = 0;
+        $education_level = NULL;
+
+        if (isset($current_user_fee_details)) {
+          // $combination = $student_gender;
+          $gender_priorities = [
+            'boy' => ['boys', 'co-ed'],
+            'girl' => ['girls', 'co-ed'],
+            'transgender' => ['co-ed', 'boys', 'girls'],
+          ];
```

Comment:
Can't we fetch this from the rte_mis_school configuration?

Reply:
These priorities are not stored in school config, we have defined this for this particular task only.

Reply:
@kunal-sin Should we store it in config and then use this?

Reply:
`boys`, ``co-ed, ``girls this we can get from the config, right?

Reply:
Are we talking about the label here?

Reply:
done.